README.md

```diff
@@ -393,10 +393,26 @@ Hanteer onderstaande conventies voor berichten die JSON als payload formaat hebb
 ### Datums en timestamps
 
 Formatteer datums en timestamps steeds volgens [RFC3339](https://www.ietf.org/rfc/rfc3339.txt). JSON definieert immers geen standaard formaat voor datums en timestamps.
+
+> RFC3339 volgt de ISO 8601 standaard maar heeft enkele optimamisaties voor het internet en machine to machine communicatie. Bijgevolg is deze ideaal voor REST API's)
+
 ```prettyprint
 "finishedAt" : "2012-01-01T01:00:00+01:00"
 ```
-Het is aangewezen om de timezone "+01:00" te gebruiken maar "2012-01-01T00:00:00Z" is ook mogelijk.
+
+Hier enkele basis regels voor het gebruik van datum en/of tijd in jouw API:
+
+1. **Aanvaard eender welke tijdzone:** Gebruikers kunnen overal ter wereld zitten of zelfs in een buurland met een andere tijdzone. Je kan moeilijk als API bouwer gokken in welke tijdzone ze zitten
+
+2. **Werk met UTC:** Dit wil zeggen:
+    * Converteer de ontvangen tijdzone naar UTC en
+    * Bewaar in jou data storage gegevens in UTC. Dit maakt het ook makkelijker om te queryen. (Stel dat je data met een lokale tijzone bewaart en er wordt een query uitgevoerd met input vanuit een andere tijdzone, dan moet je zelf in uw query deze verschillen afhandelen.
+    * Gebruik de zulu tijdsnotatie om UTC weer te geven "2012-01-01T00:00:00Z". Door die "Z" weten we dat we UTC tijd bedoelen. "2012-01-01T00:00:00" kan evengoed een tijd zijn zonder een UTC offset.
```

Comment:
Op het TA overleg is beslist om toch met tijdzone als voorkeur formaat te gebruiken (Zulu mag ook) . Het kan nuttig zijn in sommige systemen om te weten tot welke tijdzone een aanbieder behoort (bv om agendasystemen ofzo te ontwerpen). Er moet dan wel worden verduidelijkt dat de tijdsindicatie de 'lokale' tijd is van de aanbieder (Je kan altijd de Zulu time dan berekenen)